### PR TITLE
Unpack displaced pT from OMTF at the uGMT inputs

### PR DESCRIPTION
**Pull request by dinyar**
#### PR description:

uGMT now uses the unconstrained pT from OMTF, so we need to unpack it. I've added the functionality to pack and unpack it in this PR.

### CODE REVIEW

**Comment by epalencia** [comment]
Please test

File: EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc [hunk 1]
```diff
@@ -103,6 +106,9 @@ namespace l1t {
       // input muons on links 36-71
       auto gmt_in_unp = static_pointer_cast<l1t::stage2::RegionalMuonGMTUnpacker>(
           UnpackerFactory::get()->make("stage2::RegionalMuonGMTUnpacker"));
+      if (fw >= 0x8010000) {
+        gmt_in_unp->setUseEmtfDisplacementInfo();
```
Comment (missirol) [inline EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc:110]: Out of curiosity: why is this `Emtf` and not `Omtf` ?

Comment (dinyar) [reply EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc:110]: Hi @missirol,

you're absolutely right, this is a bug. Apologies. Will commit the fix either today or tomorrow.

Cheers,
Dinyar

Comment (dinyar) [reply EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc]: Committed a fix

Comment (missirol) [reply EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc]: Thanks !

**Comment by aloeliger** [comment]
+l1
- @dinyar We'll sign the backport too, I think it's okay by ORP, but they would potentially like to package it with: https://github.com/cms-sw/cmssw/pull/44394 for next week, which I am just now being made aware of. They want an L1 opinion on the unpacker, and the L1T opinion on scouting and unpackers, especially together, is unofficially whatever your opinion is. If you could take a look at that it would help us out.

**Pull request merged**